Userland/Applications/Escalator/EscalatorWindow.cpp

```diff
@@ -36,8 +37,8 @@ EscalatorWindow::EscalatorWindow(StringView executable, Vector<StringView> argum
     set_resizable(false);
     set_minimizable(false);
 
-    auto main_widget = set_main_widget<GUI::Widget>();
-    main_widget->load_from_gml(escalator_gml).release_value_but_fixme_should_propagate_errors();
+    auto main_widget = MUST(Escalator::MainWidget::try_create());
```

Comment:
This should still use `.release_value_but_fixme_should_propagate_errors()` instead of `MUST()`.